src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -25,142 +26,84 @@
 	@Autowired
 	private PersonRepository dao;
 	
-	@Autowired
-	private PersonBatchLogic personBatchLogic;
-
-	
 	@Override
-	public List<Person> getPersonByFirstName(String firstName) {
+	public Page<Person> getPersonByFirstName(Pageable pageable, String firstName) {
 		
-		return dao.findAllByFirstName(firstName);
+		return dao.findAllByFirstNameIgnoreCase(pageable, firstName);
 
 	}
 
 	@Override
-	public List<Person> getPersonByLastName(String lastName) {
+	public Page<Person> getPersonByLastName(Pageable pageable, String lastName) {
 		
-		return dao.findAllByLastName(lastName);
+		return dao.findAllByLastNameIgnoreCase(pageable, lastName);
 	}
 
 	@Override
-	public List<Person> getAllTrainees() {
+	public Page<Person> getAllTrainees(Pageable pageable) {
 		PersonRole personRole = new PersonRole(1, "Trainee");
-		return dao.findAllByPersonRole(personRole);
+		return dao.findAllByPersonRole(pageable, personRole);
 
 	}
 
 	@Override
-	public List<Person> getAllTrainers() {
+	public Page<Person> getAllTrainers(Pageable pageable) {
 		PersonRole personRole = new PersonRole(2, "Trainer");
-		return dao.findAllByPersonRole(personRole);
+		return dao.findAllByPersonRole(pageable, personRole);
 
 	}
 
 	@Override
 	public Person getPersonById(int id)  {
 		
-		BasicConfigurator.configure();
-		
 		Person p = null;
 		
 		try {
-			 
-			p = dao.getOne(id);
-			IntEvalLogger.LOGGER.info("This is my P: "+p);
-			 
-		 } catch (EntityNotFoundException e) {		
-			 PersonRole newPersonRole = new PersonRole(0, "Not a role");
-			 p = new Person("Persondoes","Notexist", newPersonRole);
-			 p.setId(0);
-			 IntEvalLogger.LOGGER.info("setting P to new person");
-			 return p;
-		 }
-		 return p;
+			
+			p = dao.findOne(id);
+			
+		} catch (EntityNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1166)